public/app/api/createBaseQuery.ts

```diff
@@ -9,7 +9,7 @@ interface RequestOptions extends BackendSrvRequest {
   body?: BackendSrvRequest['data'];
 }
 
-export function createBaseQuery({ baseURL }: { baseURL: string }): BaseQueryFn<RequestOptions, unknown, Error> {
+export function createBaseQuery({ baseURL }: { baseURL: string }): BaseQueryFn<RequestOptions> {
```

Comment:
could still type the error return type as `Error | unknown`... debatable whether it's worth it? 🤔

Reply:
Ideally this'd return `Error` type if `manageErrors` is not there and unknown otherwise, but the types for that become too complicated. I'll see if I can figure out smth simpler.